jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>"However, as real-world data is noisy, plotting the unspliced against spliced counts does not recover the expected almond shape. Instead, the data needs to be smoothed first."</p><p> </p><p>You might be able to cite something here ideally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>We can cite La Manno <em>et al.</em>, I guess. Could also just show it in a figure as I did <a href="https://www.biorxiv.org/content/10.1101/2021.12.22.473434v1.full.pdf" target="_blank">here</a> in Figure 3.</p>

Reply:
Think that a citation is good enough. Whatever you prefer though of course.